documentation/polymer-templates/tutorial-template-basic.asciidoc

```diff
@@ -47,100 +78,65 @@ class HelloWorld extends PolymerElement {
+== Creating the Java Template Class
 
+=== LitElement
 
-== Working on the Server Side
-
-=== Creating the Java Template Class
-
-To use the client-side JavaScript template on the server side, you need to create an associated Java class that extends the `PolymerTemplate` class.
+To use the client-side LitElement template on the server side, you need to create an associated Java class that extends the `LitTemplate` class.
 
 *Example*: Creating the `HelloWorld` Java template class.
 
 [source,java]
 ----
 @Tag("hello-world")
-@NpmPackage(value = "@polymer/paper-input", version = "3.0.2")
+@NpmPackage(value = "lit-element", version = "2.1.0")
 @JsModule("./src/hello-world.js")
-public class HelloWorld extends PolymerTemplate<HelloWorldModel> {
-    private static final String EMPTY_NAME_GREETING = "Please enter your name";
+public class HelloWorld extends LitTemplate {
 
     /**
      * Creates the hello world template.
      */
     public HelloWorld() {
-        setId("template");
-        getModel().setGreeting(EMPTY_NAME_GREETING);
-    }
-
-    @EventHandler
-    private void sayHello() {
-        // Called from the template click handler
-        String userInput = getModel().getUserInput();
-        if (userInput == null || userInput.isEmpty()) {
-            getModel().setGreeting(EMPTY_NAME_GREETING);
-        } else {
-            getModel().setGreeting(String.format("Hello %s!", userInput));
-        }
     }
 }
 ----
-* The `@Tag` annotation name matches the return value of the `is()` function (static getter) in the JavaScript template. This ensures that the tag name is the same on the server and the client.
+* The `@Tag` annotation name matches from the JavaScript template. This ensures that the tag name is the same on the server and the client.
 * The `@JsModule` annotation binds the Java class to the `hello-world.js` template class by specifying the relative path to the JavaScript module in the `frontend` folder in the project root. You can import multiple JavaScript resources using the `@JsModule` annotation, if needed.
-* The `@NpmPackage` annotation declares a dependency to the `paper-input` npm package: `@polymer/paper-input 3.0.2`. This annotation can be used to declare a dependency to any npm package.
-* The `@EventHandler` annotation defines the event handler called by the template click hander. Specifically, it defines the `sayHello` method that is called from the client side and triggered by a `helloButton` button click. The method name is used to map function calls between the JavaScript template and the Java class.
+* The `@NpmPackage` annotation declares a dependency to the `lit-element` npm package. This annotation can be used to declare a dependency to any npm package.
 
-=== Creating a Model Class
+=== PolymerTemplate
 
-Our example also needs a model class.
+To use the client-side Polymer template on the server side, you need to create an associated Java class that extends the `PolymerTemplate` class.
```

Comment:
Are we fine with sweeping `HelloWorldModel` under the carpet here? Even if the concept is deprecated as a reader it would feel reassuring to read something like "The `HelloWorldModel` class defines the client and server side data binding (in this case empty)." since this is the first section with code examples I am likely to read in this chapter.

Reply:
This is valid comment @joheriks. @Peppe and me agreed to remove any mentions of data binding and have only model stub, similar to what is generated by Designer.

Jens, can you comment on those? What is you vision as well? I can add the short summary, but data binding is kinda advanced topic.